pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -70,20 +71,11 @@ def load_inference_result(self) -> Mapping[str, Any]:
         : dict
             Dictionary of fit result information.
         """
-        if self._mle_fit_result is not None:
-            return self._mle_fit_result
-        return pipeline.load_inference_result(self.region)
+        return self._mle_fit_result
 
     def load_ensemble_results(self) -> Optional[dict]:
```

Comment:
should we change these to just be read only properties, now that they're not loading: 
```
@property
def ensemble_results(self):
    return self._ensemble_results
```

Reply:
Renamed a bunch of these. I'm still not sold on the benefit of making a function call look like an attribute access.